### PR TITLE
Allow public ingress from anywhere to ports 80 and 443

### LINKED ISSUE #188: LB subnet shouldn't allow traffic from internet on all ports
Load Balancer subnet allows traffic from internet on all ports - Can this be addressed? Would be better to only allow 80 and 443 as default. 

Thanks for looking into this. You all have delivered an awesome solution.

### PR DESCRIPTION
**Pull request by noeldcosta**
Closes #188 

Signed-off-by: Noel Dcosta <noel.dcosta@oracle.com>

### CODE REVIEW

**Comment by hyder** [comment]
Instead of hardcoding 80 & 443, what do you think of setting these 2 as default but let users to specify their own list of ports?

+ @markxnelson

**Comment by noeldcosta** [comment]
have a look at https://github.com/oracle-terraform-modules/terraform-oci-oke/compare/master...noeldcosta:188-fixes

**Comment by noeldcosta** [comment]
@hyder the change set has been updated with support for a custom list of ports

**Review by hyder — CHANGES_REQUESTED** [review]
Please also document the changes in:
1. Configuration (under Configure OKE Load Balancer parameters)
2. Terraform Options

Lastly, add them in the terraform.tfvars.example file

**Comment by noeldcosta** [comment]
Documentation has been updated.

File: docs/terraformoptions.adoc [hunk 1]
```diff
@@ -554,6 +554,11 @@ Refer to {uri-topology}[topology] for more thorough examples.
 |internal/public
 |public
 
+|`public_lb_ports`
+|The list of destination ports to allow for public ingress.
+|List of ports.
```
Comment (hyder) [inline docs/terraformoptions.adoc:559]: 3rd column should be an example [80,443]

File: modules/okenetwork/variables.tf [hunk 2]
```diff
@@ -35,8 +35,9 @@ variable "lb_subnet_type" {
   type = string
 }
 
+variable "public_lb_ports" {}
```
Comment (hyder) [inline modules/okenetwork/variables.tf:38]: please set type list(string)

Comment (noeldcosta) [reply modules/okenetwork/variables.tf]: Have set it to below to match the variable definition in variables.tf

variable "public_lb_ports" {
  type = list(number)
}

**Review by hyder — APPROVED** [review]

**Pull request merged**